src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -179,14 +198,31 @@ export default class rd2EntryForm extends LightningElement {
     /*******************************************************************************
     * @description Modify edit modal header
     */
-    modidifyEditHeader() {
+    modifyEditHeader() {
         this.header = editHeaderLabel + ' ' + this.record.fields.Name.value;
     }
 
+    /*******************************************************************************
+    * @description Get today's date of month
+    */
+    getTodayDateOfMonth() {
+        let todayDate = new Date().getDate().toString();
+        
+        let convertedPicklist = this.dayOfMonthPicklistValue.find(value => {
```

Comment:
- If today is 31st, what will default be, "Last day in month" label (not Last_Day API name)?
- Let's specify in the description what the output is, a date, a day?

Reply:
Lightning-InputField will automatically takes care of the api to label conversion.